org-code-javabuilder/protocol/src/main/java/org/code/protocol/JavabuilderThrowableMessageUtils.java

```diff
@@ -93,4 +105,12 @@ public static String getUserFacingStackTraceString(Throwable cause) {
   private static String getExceptionMessageString(Throwable cause) {
     return cause.toString();
   }
+
+  private static String trimDetail(String detailMessage, int charLimit) {
+    String trimmedMessage = detailMessage;
+    if (detailMessage.length() > charLimit) {
+      trimmedMessage = trimmedMessage.substring(0, charLimit);
+    }
+    return trimmedMessage;
```

Comment:
nit: could potentially simplify this to `return detailMessage.substring(0, Math.min(charLimit, detailMessage.length()))`